controllers/disruption_controller.go

```diff
@@ -1043,15 +1045,29 @@ func (r *DisruptionReconciler) ReportMetrics(ctx context.Context) {
 			chaosPodsCount += len(chaosPods)
 
 			r.handleMetricSinkError(r.MetricsSink.MetricDisruptionOngoingDuration(time.Since(d.ObjectMeta.CreationTimestamp.Time), []string{"disruptionName:" + d.Name, "namespace:" + d.Namespace}))
+
+			if count, ok := namespaces[d.Namespace]; ok {
+				namespaces[d.Namespace] = count + 1
+			} else {
+				namespaces[d.Namespace] = 1
+			}
 		}
 
 		// send metrics
 		if err := r.MetricsSink.MetricStuckOnRemovalGauge(float64(stuckOnRemoval)); err != nil {
 			r.BaseLog.Errorw("error sending stuck_on_removal_total metric", "error", err)
 		}
 
-		if err := r.MetricsSink.MetricDisruptionsGauge(float64(len(l.Items))); err != nil {
-			r.BaseLog.Errorw("error sending disruptions.gauge metric", "error", err)
+		if len(namespaces) > 0 {
```

Comment:
I've tested and this works in the case of 0 disruptions, disruptions in 1 namespace, and disruptions in many namespaces

Reply:
Why do we need to make the verification here? If I'm not mistaken namespaces can't be nil no?